libs/libssh/Makefile

```diff
@@ -51,11 +51,12 @@ CMAKE_OPTIONS = \
 	-DWITH_GSSAPI:BOOL=OFF \
 	-DWITH_LIBZ:BOOL=ON \
 	-DWITH_NACL:BOOL=OFF \
-	-DWITH_PCAP:BOOL=OFF \
+	-DWITH_PCAP:BOOL=ON \
 	-DWITH_SERVER:BOOL=ON \
 	-DWITH_SFTP:BOOL=ON \
 	-DWITH_SSH1:BOOL=OFF \
 	-DWITH_TESTING:BOOL=OFF \
+	-DWITH_STATIC_LIB:BOOL=ON \
```

Comment:
Take a look at `Package/libssh/install`.  I am afraid it will indeed ship the built static library.  I don't think it's valid request to enable static build for all just for debugging purposes.  That should be handled locally.